tests/nnstreamer_filter_python/runTest.sh

```diff
@@ -44,11 +44,18 @@ fi
 FRAMEWORK="python2"
 # This symlink is necessary only for testcases; when installed, symlinks will be made
 pushd ../../build/ext/nnstreamer/tensor_filter
-ln -s nnstreamer_python2.so nnstreamer_python.so
+TEST_PYTHONPATH=${FRAMEWORK}_pymodule
```

Comment:
Here is `python2_pymodule` while similar path in `packaging/run_unittests_binaries.sh:23` is `python[23]_module` . IMO this should be the same if there isn't any reason.

Reply:
It would be good to use same path but not should be. These two scripts are not invoked in the same context. Note that runTest.sh is invoked by SSAT.